From 743c26c982a97bd5d8deb06c16a53b28ab92129d Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 30 Nov 2023 08:15:08 +0100
Subject: [PATCH] luna-sysservice: Fix permissions for telephony

webos-telephonyd uses telephony.management and telephony.query, so give permissions to this too.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.webos.service.systemservice.perm.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.systemservice.perm.json b/files/sysbus/com.webos.service.systemservice.perm.json
index bef94d8..d30775a 100644
--- a/files/sysbus/com.webos.service.systemservice.perm.json
+++ b/files/sysbus/com.webos.service.systemservice.perm.json
@@ -1,6 +1,6 @@
 {
 	"com.webos.service.systemservice":[
 		 "settings.query", "application.launcher", "systemsettings.management",
-		 "networkconnection.query"
+		 "networkconnection.query", "telephony.management", "telephony.query" 
 	]
 }
